hugegraph-core/src/main/java/com/baidu/hugegraph/backend/store/raft/StoreCommandRequestProcessor.java

```diff
@@ -75,9 +68,9 @@ public String interest() {
     }
 
     private StoreCommand parseStoreCommand(StoreCommandRequest request) {
-        byte actionByte = (byte) request.getAction().getNumber();
-        StoreAction action = StoreAction.fromCode(actionByte);
+        StoreType type = request.getType();
+        StoreAction action = request.getAction();
         byte[] data = request.getData().toByteArray();
-        return new StoreCommand(action, data);
+        return new StoreCommand(type, action, data);
```

Comment:
if request.getData() returns ByteBuffer, just wrap it with BytesBuffer then pass into  StoreCommand

Reply:
no, it return com.google.protobuf.ByteString